serverless.yml

```diff
@@ -66,15 +67,21 @@ functions:
       - http: 'GET /art'
         cors: true
 
+  searchArt:
+    handler: index.handler
+    events:
+      - http: 'POST /search/art'
+        cors: true
+
   createArt:
     handler: index.handler
     events:
       - http: 'POST /art'
         cors: true
-    authorizer:
-        type: COGNITO_USER_POOLS
-        authorizerId:
-            Ref: ApiGatewayAuthorizer
+    #authorizer:
+    #    type: COGNITO_USER_POOLS
+    #    authorizerId:
+    #        Ref: ApiGatewayAuthorizer
 
 resources:
   Resources:
```

Comment:
after a successful migration we will need to update the serverless to blow away the dynamo

Reply:
Agreed once we have it updated we blow again the dynamo